devbox.json

```diff
@@ -0,0 +1,17 @@
+{
+  "packages": [
+    "nodejs@18.18.2",
+    "corepack@18.18.2",
+    "docker@latest"
+  ],
+  "shell": {
+    "init_hook": [
+      "echo 'Welcome to devbox!' > /dev/null"
+    ],
+    "scripts": {
+      "test": [
+        "echo \"Error: no test specified\" && exit 1"
```

Comment:
Would it make sense to specify the unit or integration tests here?

Reply:
I'd probably leave it out and remove the `scripts` section. We will continue to use `npm` to run tests. 1 singular place to encapsulate the needed scripts.